### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by evidencem** [dedoc/scramble#504] [Q&A] [answered]
This is an awesome package! Is it possible to add image links in the doc blocks and have them render in the documentation? :eyes: 

I would like to able to show flow chart diagrams to users.

**Comment by romalytvynenko** [comment, marked as answer]
@evidencem Scramble uses Stoplight Elements for UI, so you can just try inserting image like you would in a markdown. If it doesn't work, I don't think it supports this.

I'm working on new UI for Scramble, I will make sure to add the support

**Reply by evidencem** [reply]
@romalytvynenko Thank you very much for the solution. It works brilliantly!

**Comment by nerisonpitogo** [comment]
@romalytvynenko The current UI is already fantastic, and now you're creating a new one? I'm excited to see it!
